pandas/io/tests/parser/test_read_fwf.py

```diff
@@ -345,3 +345,41 @@ def test_variable_width_unicode(self):
                             header=None, encoding='utf8')
         tm.assert_frame_equal(expected, read_fwf(
             BytesIO(test.encode('utf8')), header=None, encoding='utf8'))
+
```

Comment:
if it uses ``self.read_csv`` it *should*, but not sure fwf is really all that well tested.